src/audio/copier/copier.c

```diff
@@ -1688,13 +1695,18 @@ static int copier_get_attribute(struct comp_dev *dev, uint32_t type, void *value
 static int copier_position(struct comp_dev *dev, struct sof_ipc_stream_posn *posn)
 {
 	struct copier_data *cd = comp_get_drvdata(dev);
+	int ret = 0;
 
 	/* Exit if no endpoints */
 	if (!cd->endpoint_num)
 		return -EINVAL;
 
+	if (dev->ipc_config.type == SOF_COMP_HOST)
+		host_zephyr_position(cd->hd, posn);
```

Comment:
yes, when I add, I just want to make it aligned with other API, it can be removed like processed data, please check processed data, if you are ok, I will remove both.